.github/workflows/pr_cherrypick_poststep.yml

```diff
@@ -0,0 +1,58 @@
+name: PostCherryPick
+on:
+  pull_request_target:
+    types:
+    - closed
+    branches:
+    - 202205
+    - 202111
+    - 202106
+    - 202012
+    - 201911
+    - 201811
+
+jobs:
+  post_cherry_pick:
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'automerge')
```

Comment:
Looks like the condition is not enough. We need to add a new tag to indicate cherry-pick from master.
Some other PRs not cherry-pick from master, for example, the pr to upgrade the versions,  should not trigger the run, right?

Reply:
It is related with automerge actions.
If we use another label, we need another automerge action. It will not have less action jobs.